### ISSUE TITLE
[Feature Request] Move currency dropdown box to top of page instead of bottom

### ISSUE THREAD

**Issue by epichosts** [FOSSBilling/FOSSBilling#1114]
**Is your feature request related to a problem? Please describe.**
Currently the dropdown box for the currencies is at the bottom of a product/order, like so: ![01](https://user-images.githubusercontent.com/25933763/232567137-ea07b89c-0aeb-48b2-8373-aa4c033712f4.png)

Which makes it very difficult for the general user to find or even know exists.

**Describe the solution you'd like**
To have the dropdown box moved to the top of the page, like so: ![01](https://user-images.githubusercontent.com/25933763/232567478-8e367069-3c9a-4537-a09c-98ee75cee753.png)

Or anywhere else on the page that would have better visibility to clients.

**Describe alternatives you've considered**

I have moved it to the top myself, but updates will overwrite this and I think it would be better to have it as an official update.

**Additional context**
N/A

**Comment by Ron-Zarbyte** [comment]
It can be moved to the top with this code in `mod_order_index.html.twig`:

```
<header>
    <h2>{{ 'Select 44Product'|trans }}</h2>
    <p>
        {{ 'Choose products we offer for selling'|trans }}
        {% include 'mod_orderbutton_currency.html.twig' %}
    </p>
</header>
```

Then hide the bottom one by removing this from `mod_orderbutton_index.html.twig`:
```
{% include 'mod_orderbutton_currency.html.twig' %}
```


The currency selector is functional with this edit, however it introduces a new bug, so it is not suitable to submit a pull request with.

The bug is the top currency selector causing a full page refresh instead of of an inline refresh.

Steps to reproduce:

1. Visit order form and select a product with pricing options.
2. When the prices are displaced, change the currency.
3. The page will now fully refresh and require you to select the product again.

When using the bottom currency selector, it triggers a refresh of just the product content and updates the prices without needing to select the product again.

The header where the top currency selector would be is located outside of the `orderbutton` div, so it does not get handled by the current page refresh logic. This would need to be looked at a little more deeply to make a better solution.

tl;dr: more than just moving an element, and I am not familiar enough with frontend structure yet to address it myself. I will come back to this in the future if someone else does not beat me to it.

**Comment by Anuril** [comment]
This is probabaly also impacted by #1602, right @BelleNottelling?
